[Logs] Add some performance recommendations for the log threshold rule type (#804)
elastic/observability-docs: docs/en/observability/logs-threshold-alert.asciidoc

[role="screenshot"]
image::images/alert-action-types.png[Alert action types]

[discrete]
[[performance-considerations]]
=== Performance considerations

> When setting a group by we highly recommend using the "more than" comparator for your threshold if possible. This allows our queries to apply eager filtering, which can lead to significant performance improvements. If this isn't possible we suggest using a group by field with the lowest cardinality (number of possibilities) possible.

Reviewer: ```suggestion
When setting a *group by*, we recommend using the *more than* comparator for your threshold—this allows our queries to apply eager filtering, leading to significant performance improvements. Otherwise, we suggest using a *group by* field with the lowest cardinality (number of possibilities).
```
